class/class.midtrans-gateway-notif-handler.php

```diff
@@ -72,8 +72,16 @@ public function midtrans_response() {
       $raw_notification = $this->earlyResponse();
       // Handle pdf url update
       $this->handlePendingPaymentPdfUrlUpdate();
+      // Get WooCommerce order
+      $wcorder = wc_get_order( $raw_notification['order_id'] );
+      // exit if the order id doesn't exist in WooCommerce dashboard
+      if (!$wcorder) {
+        WC_Midtrans_Logger::log( 'Can\'t find order id' . $raw_notification['order_id'] . ' on WooCommerce dashboard', 'midtrans-error' );
+        error_log('Can\'t find order id' . $raw_notification['order_id'] . ' on WooCommerce dashboard');
```

Comment:
Is this `error_log` line expected or forgot to be removed? because there is already `WC_Midtrans_Logger::log` above it.

Reply:
oh right! my bad thanks for checking

Reply:
done https://github.com/veritrans/SNAP-Woocommerce/pull/42/commits/a8dc324bc7e4cd4e32feb056078607fd8799f40a